peer/network_test.go

```diff
@@ -30,6 +30,10 @@ import (
 	"github.com/ava-labs/avalanchego/version"
 )
 
+const (
+	codecVersion uint16 = 0
+)
```

Comment:
nit: since this only used for tests, perhaps let's rename it to `testCodecVersion` to avoid any confusion?

Also would it be possible to use another version, like `1`? Personal opinion, it's nice to test with non-zero values to verify nothing was forgotten (since default = zero value)